cli/main/main.go

```diff
@@ -14,7 +14,7 @@ func main() {
 	factory := &dockerApp.ProjectFactory{}
 
 	app := cli.NewApp()
-	app.Name = "docker-compose"
+	app.Name = "libcompose-cli"
 	app.Usage = "Fast, isolated environments using Docker."
```

Comment:
Maybe this description should also change to something like "Command-line interface to libcompose"